### PR TITLE
fix: TR-3034. Change icons for RTL mode

### PR DESCRIPTION
**Pull request by svoronuk**
Related to: https://oat-sa.atlassian.net/browse/TR-3034

How to test:
1. Add test taker with Arabic language
2. Publish test and assign created user
3. Check the icons styles according to ACs

### CODE REVIEW

**Review by olga-kulish — APPROVED** [review]
Works well 

- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: views/scss/delivery.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @import "inc/bootstrap";
-@import "inc/rtl";
+
```
Comment (oatymart) [inline views/scss/delivery.scss:2]: Why did you remove the `inc/rtl` import? Seems like that contains [a file](https://github.com/oat-sa/extension-tao-delivery/blob/master/views/scss/inc/rtl/_header.scss#L26) which affects very much the header layout in RTL mode, including the menu items and logout icon (including changes applied [very recently](https://github.com/oat-sa/extension-tao-delivery/pull/522))...

However, don't trust me on this - better to check with BA.

**Review by oatymart — COMMENTED** [review]
The play icon flip is present. Please check which way the page header should flow, it's not mentioned anywhere.

File: views/scss/delivery.scss (hunk 1 above)
Comment (svoronuk) [reply views/scss/delivery.scss:2]: It was made by accident. I'll fix this.

File: views/scss/delivery.scss [hunk 2]
```diff
@@ -67,5 +68,12 @@ html {
...
     }
-
+    &[dir=rtl] {
+        .test-listing{
+            .icon-continue, .icon-play {
+                transform: rotate(180deg);
+                display: inline-block;
+            }
+        }
+    }
```
Comment (oatymart) [inline views/scss/delivery.scss:78]: The solution works. However, I'm still not sure your code is in the right place. Seems to me, previous developers intended that all `rtl` styles should go inside `inc/rtl` folder. I would suggest to create a file `inc/rtl/_content.scss` and put the block inside it. WDYT?

**Review by oatymart — CHANGES_REQUESTED** [review]
I had another thought about where the code should live...

File: views/scss/delivery.scss (hunk 2 above)
Comment (svoronuk) [reply views/scss/delivery.scss:78]: Sounds reasonable. I'll modify it.

**Review by oatymart — APPROVED** [review]
Looks good now!

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**